network/src/protocols/rpc/mod.rs

```diff
@@ -492,10 +495,14 @@ async fn handle_outbound_rpc_inner(
         peer_id.short_str()
     );
     let response = response_rx.await?;
+    let latency = timer.stop_and_record();
     trace!(
-        "Received response for request_id {} from peer: {:?}",
+        "Received response for request_id {} from peer: {:?} \
+        with {:.6} seconds of latency. Request protocol_id: {}",
```

Comment:
were you able to verify this?

Reply:
Actually not yet.. Do you have any suggestions about a good way to test it?